aerofiles/seeyou/reader.py

```diff
@@ -4,13 +4,51 @@
 from aerofiles.errors import ParserError
 
 
-RE_COUNTRY = re.compile(r'^([\w]{2})?$', re.I)
-RE_LATITUDE = re.compile(r'^([\d]{2})([\d]{2}\.[\d]{3})([NS])$', re.I)
-RE_LONGITUDE = re.compile(r'^([\d]{3})([\d]{2}\.[\d]{3})([EW])$', re.I)
-RE_ELEVATION = re.compile(r'^(-?[\d]*(?:\.[\d]+)?)\s?(m|ft)?$', re.I)
-RE_RUNWAY_LENGTH = re.compile(r'^(?:([\d]+(?:\.[\d]+)?)\s?(ml|nm|m)?)?$', re.I)
-RE_FREQUENCY = re.compile(r'^1[\d]{2}\.[\d]+?$')
-RE_DISTANCE = re.compile(r'^(-?[\d]*(?:\.[\d]+)?)\s?(m|ft|km|ml|nm)?$', re.I)
+RE_COUNTRY = re.compile(r"^([\w]{2})?$", re.I)
```

Comment:
any reason for all these quote style changes?

Reply:
Auto formatting by black, I will see if I can change the style for this project.

Reply:
using black is probably fine, but I'd prefer if we would change it in a dedicated PR then and integrate it directly with the CI pipeline to avoid regressions